pragma/file.go

```diff
@@ -108,8 +110,23 @@ func (f *File) extractPragmas() {
 				continue
 			}
 
+			// Previous line also has code
+			if !hasPrefixes(previousLine, f.CommentPrefix) {
```

Comment:
white space will also be treated as previous new-line having issue no?

Reply:
Yup, the line is already trimmed.

Reply:
cool, makes sense. But what happens if it's a blank line?

Reply:
```
code();

// [CODE]
code();
```

Reply:
I think it should be fine but just confirm that much I think that's the only confusing thing otherwise lgtm

Reply:
Sure.

Reply:
It works as expected, also added a test case for the same.